OrbitGl/TopDownView.h

```diff
@@ -7,10 +7,10 @@
 
 #include <unordered_map>
 
-#include "SamplingProfiler.h"
+#include "CaptureData.h"
 #include "absl/container/node_hash_map.h"
 
-class [[nodiscard]] TopDownNode {
```

Comment:
Just out of curiosity. Why remove those?

Reply:
They confused clang-format (as Dimitry has noted correctly), so I removed them.

Reply:
okay